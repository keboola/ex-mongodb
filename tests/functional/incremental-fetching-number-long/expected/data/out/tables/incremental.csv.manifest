{"incremental":true,"columns":["id","numberLong","numberLongInObject"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"numberLong":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"numberLongInObject":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}]}}