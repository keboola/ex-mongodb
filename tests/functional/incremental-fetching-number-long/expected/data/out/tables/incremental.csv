"id","numberLong","numberLongInObject"
"5716054bee6e764c94fa8213","1050845886852","1050845886852"
"5716054bee6e764c94fa8181","2000000000000","2000000000000"
"5716054bee6e764c94fa7ddd","2090845886852","2090845886852"